lib/galaxy/webapps/galaxy/controllers/dataset.py

```diff
@@ -519,7 +519,7 @@ def imp( self, trans, dataset_id=None, **kwd ):
         """ Import another user's dataset via a shared URL; dataset is added to user's current history. """
         # Set referer message.
         referer = trans.request.referer
-        if referer is not "":
+        if referer is not None:
```

Comment:
If it would make sense to remain compatible with the old version of WebOb, then this could be changed to:
```
        if referer:
```
and similarly below.